scheduler/repositories/stores.py

```diff
@@ -80,3 +81,21 @@ def get_item_by_hash(self, scheduler_id: str, item_hash: str) -> Optional[models
     @abc.abstractmethod
     def get_items_by_scheduler_id(self, scheduler_id: str) -> List[models.PrioritizedItem]:
         raise NotImplementedError
+
+
+class JobStorer(abc.ABC):
+    def __init__(self) -> None:
+        self.logger: logging.Logger = logging.getLogger(__name__)
```

Comment:
Personal/ suggestion:
```suggestion
        self.logger: logging.Logger = logging.getLogger(self.__name__)
```

This will give the concrete name of the implemented class rather than the module name

Reply:
I'll have a look if I've done this in the other classes as well.